v4/export/sql_type.go

```diff
@@ -140,50 +135,84 @@ func (r RowReceiverArr) ReportSize() uint64 {
 }
 func (r RowReceiverArr) ToString(escapeBackslash bool) string {
 	var sb strings.Builder
```

Comment:
maybe you can pass the `bytes.Buffer` explictily to reduce more allocation.

Reply:
function `ToString` is not used any more. Now `write_util` uses `WriteToBuffer` to write to `bytes.buffer` directly and to avoid allocation.